src/autocomplete/AutocompleteView.js

```diff
@@ -46,7 +46,8 @@ export default function AutocompleteView(props: Props): Node {
 
   const { lastWordPrefix, filter } = getAutocompleteFilter(text, selection);
   const AutocompleteComponent = prefixToComponent[lastWordPrefix];
-  const shouldShow = isFocused && !!AutocompleteComponent && filter.length > 0;
+  const shouldShow =
+    (isFocused && !!AutocompleteComponent && filter.length > 0) || lastWordPrefix === '@';
```

Comment:
I don't understand this change. 🙂 Could you please talk about what problem this change is meant to solve, and why it's the right way to solve it?

I notice that, before this change, `lastWordPrefix === '@'` is already a factor in computing the value of `shouldShow`. Note that if `lastWordPrefix` is `'@'`, then `AutocompleteComponent` will be `PeopleAutocomplete`, and therefore `!!AutocompleteComponent` will be `true`.

Reply:
Thanks, for your point on view about the `lastWordPrefix`, I see the err in my code